examples/basic/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `cp` or `cloudposse`)"
```

Comment:
let's not use `cp` or `cloudposse` in examples (people try to use it verbatim and have issues with naming collisions)

description = "Namespace (e.g. `eg`)"

Reply:
i've change description